tests/functional/commands/test_hunt_command.py

```diff
@@ -35,3 +35,26 @@ def test_no_name(capsys):
     captured = capsys.readouterr().err
 
     assert captured == "Error: No name provided for the experiment.\n"
+
+
+def test_user_script_crash(capfd):
+    """Checks that the Traceback of a crashing user script is output to stderr"""
+
+    orion.core.cli.main(
```

Comment:
The test if failing because the script is not found. Pytest is executed from the root so it cannot find this file at the root. You can use monkeypatch to change the working dir temporarily: https://github.com/Epistimio/orion/blob/develop/tests/functional/commands/conftest.py#L61

Note, the first test of this module is not crashing because it has the fixture `one_experiment` which does this change of working directory.